framework/db/Connection.php

```diff
@@ -120,7 +120,8 @@
  * read-only.
  * @property Schema $schema The schema information for the database opened by this connection. This property
  * is read-only.
- * @property Connection $slave The currently active slave connection. Null is returned if there is slave
+ * @property Connection $master The currently active master connection. Null is returned if there is no master
```

Comment:
no, the annotation should be on the getter. it will be shown in the apidocs correctly.